scripts/macos/log_collection.sh

```diff
@@ -113,7 +113,22 @@ fi
 
 ## Only run if a user is actually logged in
 if [[ $localuser ]]; then
+
+    ## pull additional patch management information
     sudo -u $localuser defaults read com.github.macadmins.Nudge.plist > $baseDir/com.github.macadmins.Nudge.plist 2>&1
+
+    ## list jumpcloud services currently running on the system
+    sudo -u $localuser launchctl print system | grep -i 'jumpcloud' > $baseDir/systemInfo/activeJumpCloudServices.txt
+
+    ## list JumpCloud Device Certificate
+    if sudo -u "$localuser" security find-certificate -c "JumpCloud Device Trust Certificate" -p > /dev/null 2>&1; then
+        sudo -u $localuser security find-certificate -c "JumpCloud Device Trust Certificate" -p | openssl x509  -text > $baseDir/systemInfo/deviceCert.txt
```

Comment:
Just to check. what's the output on your device when you run `which openssl`? By default macOS uses libressl and openssl is an alias to that distribution of openssl by default. That version is in /usr/bin/openssl 
![Screenshot 2025-03-07 at 1 21 05 PM](https://github.com/user-attachments/assets/c7a1bbf0-6676-4d0c-a242-fbe9a48dbb37)

This seems to work when I change the line to /usr/bin/openssl from just openssl.

So only suggestion here is to remove the extra space

```suggestion
        sudo -u $localuser security find-certificate -c "JumpCloud Device Trust Certificate" -p | openssl x509 -text > $baseDir/systemInfo/deviceCert.txt
```

Reply:
Sure, so my local output is: 

`/opt/homebrew/bin/openssl`

Though checking on a VM and test device I have on my end and been using, the outputs are both:

`/usr/bin/openssl`


I've got that extra space removed.